host/integration_test.go

```diff
@@ -1613,12 +1613,12 @@ func (s *IntegrationSuite) TestRateLimitBufferedEvents() {
 			// Buffered Signals
 			for i := 0; i < 100; i++ {
 				buf := new(bytes.Buffer)
-				binary.Write(buf, binary.LittleEndian, i)
+				binary.Write(buf, binary.LittleEndian, int32(i))
```

Comment:
any reason for int32 vs int64?  I think this was behaving as int64, as int is architecture-dependent and basically nobody uses 32-bit for servers

Reply:
Good point, apparently I'm living in the past :)
Updated.